src/machine/machine_atmega328p.go

```diff
@@ -0,0 +1,15 @@
+// +build avr,atmega328p
+
+package machine
+
+import "device/avr"
+
+const irq_USART0_RX = avr.IRQ_USART_RX
+
+// LED on the Arduino
+const LED Pin = 13
```

Comment:
The LED pin is board specific so should be defined in board_arduino-mega2560.go (with the appropriate build tag: `// +build arduino_mega2560`).

Reply:
Ok thanks for the clarification.  It was unclear to me what pieces went in which files.  I have seen mixtures in other files that contradicted themselves.